js/exhibit.js

```diff
@@ -152,5 +152,11 @@ $(document).ready(function() {
       $('.repos-count').html('Showing ' + index + ' of ' + total)
     });
   }
+
+  $(".dropdown-button").dropdown(
+    { hover: true }
+  );
+
+  $(".button-collapse").sideNav(function() {});
```

Comment:
you can call  `$(".button-collapse").sideNav()`  if you do not want to pass any options